packages/plans-grid-next/src/components/features-grid/index.stories.tsx

```diff
@@ -38,12 +37,36 @@ export default {
 			defaultViewport: 'LARGE',
 		},
 	},
-} as Meta;
+};
+
+export default meta;
 
-const storyDefaults = {
-	args: defaultArgs,
+type Story = StoryObj< typeof FeaturesGrid >;
+
+export const PlansFlow: Story = {
+	args: {
+		...defaultArgs,
+		intent: 'plans-default-wpcom',
+		gridPlans: [
+			mockGridPlans.free,
+			mockGridPlans.personal,
+			mockGridPlans.value,
+			mockGridPlans.business,
+			mockGridPlans.ecommerce,
+			mockGridPlans.enterprise,
+		],
+		gridPlanForSpotlight: mockGridPlans.spotlight,
+	},
 };
 
-export const FeaturesGridTest = {
-	...storyDefaults,
+PlansFlow.storyName = '/plans';
+
+export const NewsletterFlow: Story = {
+	args: {
+		...defaultArgs,
+		intent: 'plans-newsletter',
+		gridPlans: [ mockGridPlans.free, mockGridPlans.personal, mockGridPlans.value ],
```

Comment:
In product-side code, i.e. in `PlansFeaturesMain`, `gridPlans` is derived from `intent`. I'm wondering if we can also reflect this from here 🤔 

It's more of an open question that we can explore further, so no need to address it here. This is already a better step forward :)

Reply:
Thanks for the review @southp.

> In product-side code, i.e. in PlansFeaturesMain, gridPlans is derived from intent. I'm wondering if we can also reflect this from here

Yes 100%. I'm working on this change to derive `gridPlans` from `intent` in this followup PR: https://github.com/Automattic/wp-calypso/pull/89837